frontend/src/main/web/lib/stores/GlossaryStore.js

```diff
@@ -0,0 +1,522 @@
+import Dispatcher from '../dispatchers/GlossaryDispatcher';
+import assign from 'object-assign';
+import {EventEmitter} from 'events';
+import {Promise} from 'es6-promise';
+import Request from 'superagent';
+import Configs from '../constants/Configs';
+import {GlossaryActionTypes} from '../constants/ActionTypes';
+import GlossaryHelper from '../utils/GlossaryHelper'
+import StringUtils from '../utils/StringUtils'
+import DateHelpers from '../utils/DateHelper'
+import _ from 'lodash';
+
+var SIZE_PER_PAGE = 5000, CHANGE_EVENT = "change",
+    MAX_LISTENER = 100; //number of listener for GlossaryStore (default is 11)
+
+EventEmitter.prototype.setMaxListeners(MAX_LISTENER);
```

Comment:
> How can I do that?

Don't call `setMaxListeners()` on `EventEmitter.prototype`. When you make an object that inherits from `EventEmitter` you can call `this.setMaxListeners()` when it is constructed.